Examples/JavaScript/JavaScript Essentials/Objects and Arrays.ts

```diff
@@ -0,0 +1,96 @@
+//// { order: 1, compiler: { strict: false } }
+
+// JavaScript objects are collections of values wrapped up with named keys.
+
+const userAccount = {
+  name: "Kieron",
+  id: 0
+}
+
+// You can combine these to make larger, more complex data-models.
+
+const pie = {
+  type: "Apple"
+}
+
+const purchaseOrder = {
+  owner: userAccount,
+  item: pie
+}
+
+// If you use your mouse to hover over some of these words (try purchaseOrder above)
```

Comment:
on my 1440px screen, I get 76 characters per pane, which cuts off a few of these lines. Is it worthwhile to reformat to avoid this? I don't know how common 1440 and smaller widths are these days.

Reply:
Hah, great point... I'm now [on an ultra wide](https://www.dell.com/en-us/shop/dell-ultrasharp-34-curved-usb-c-monitor-u3419w/apd/210-arcl/monitors-monitor-accessories?prg=1&VEN1=12839518-5513721-AwEAAAAAAAAAAWlf&AID=5513721&cjevent=2e2fc897bd3311e9815d03f50a240614&dgc=CJ&DGSeg=DHS&cid=198375&lid=45846&acd=12309198375458460&VEN3=811705054906337422) so my experiences are _real_ far from reality nowadays. Thanks will try bump it down to 60